deploy/consts.ts

```diff
@@ -16,6 +16,7 @@ export const QUOTE_TIME_BUFFER = 3600;
 export const FILL_DEADLINE_BUFFER = 6 * 3600;
 export const ARBITRUM_MAX_SUBMISSION_COST = "10000000000000000";
 export const AZERO_GAS_PRICE = "240000000000";
+export const CIRCLE_UNINITIALIZED_DOMAIN_ID = 4294967295; // 2^32 - 1
```

Comment:
I thought we usually use -1 for uninitialized?

Reply:
We use -1 in `constants`, but `cctpDomainId` is a `number` field there so it can be negative; here's it's an unsigned type.